.gitignore

```diff
@@ -43,6 +43,7 @@ frontend/index.html
 
 # Challenge data files
 challenge_data/*
+examples/example1/evaluation_script/*
```

Comment:
I dont think we need it here. It is totally fine to push the dummy evaluation script to the repository.